tests/rt.sh

```diff
@@ -212,7 +212,7 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   QUEUE=batch
   COMPILE_QUEUE=batch
 
-#  ACCNR=fv3-cpu
+  ACCNR=coastal
```

Comment:
Please revert this change. This will not work for all users.

Reply:
@DusanJovic-NOAA Done!!!